packages/flutter_tools/lib/src/ios/device_ios.dart

```diff
@@ -560,8 +561,11 @@ String _getIOSEngineRevision(ApplicationPackage app) {
 
 Future<bool> _buildIOSXcodeProject(ApplicationPackage app, { bool buildForDevice }) async {
   if (!FileSystemEntity.isDirectorySync(app.localPath)) {
-    printError('Path "${path.absolute(app.localPath)}" does not exist.\nDid you run `flutter ios --init`?');
-    return false;
+    printTrace('Path "${path.absolute(app.localPath)}" does not exist. Initializing the Xcode project on behalf of the user.');
```

Comment:
I'd remove "on behalf of the user".  Everything we do is on behalf of the user. :)